### PR TITLE
Add missing documentation steps for setting up AFJ for RN

### PR DESCRIPTION
**Pull request by MosCD3**
Missing adding the file system package which will yield in exception when running the code in the documentation

### CODE REVIEW

File: docs/setup-react-native.md [hunk 1]
```diff
@@ -31,6 +31,16 @@ import 'react-native-get-random-values'
...
 ```
 
+6. Install react native file system and update pods
+
+```bash
+yarn add react-native-fs
+```
+
+```bash
+cd ios  and pod install
```
Comment (jakubkoci) [inline docs/setup-react-native.md:41]: What about the following command:
```suggestion
pod install --project-directory=ios/
```

User can copy-paste.

**Review by jakubkoci — COMMENTED** [review]
We will hopefully get rid of this step with a monorepo. But good update, for now, thanks 👍

**Comment by jakubkoci** [comment]
@MosCD3 Could you please fix the DCO check by singing commits?

File: docs/setup-react-native.md [hunk 2]
```diff
@@ -31,6 +31,16 @@ import 'react-native-get-random-values'
 import '@azure/core-asynciterator-polyfill'
 ```
 
+6. Install react native file system and update pods
+
+```bash
+yarn add react-native-fs
```
Comment (TimoGlastra) [inline docs/setup-react-native.md:37]: `react-native-fs` is already a dependency. Only the pod install is required

**Review by TimoGlastra — APPROVED** [review]

**Review by TimoGlastra — CHANGES_REQUESTED** [review]
.

**Pull request closed without merging**